recipe/meta.yaml

```
@@ -9,13 +9,18 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/awkward-{{ version }}.tar.gz
   sha256: d856b4a166ae054363368aed2e4a44338fec069baa4242e7d567c8323ebcc1eb
+  patches:
+    - awkward-1.0.2-singlethread.patch
 
 build:
   number: 1
-  script: {{ PYTHON }} -m pip install . -vv
+  script: CMAKE_BUILD_PARALLEL_LEVEL=1 {{ PYTHON }} -m pip install . -vvv  # [linux and ppc64le]
+  script: {{ PYTHON }} -m pip install . -vvv  # [not (linux and ppc64le)]
```

As it's not really a ppc64le issue but rather a many-core-machine issue:
```suggestion
  script:
      - export CMAKE_BUILD_PARALLEL_LEVEL=${CPU_COUNT}
      - {{ PYTHON }} -m pip install . -vvv
```

---

I'm guessing Conda-forge sets this properly, while simply detecting the core count is incorrect?

---

The problem was that awkward was using multiprocessing.cpu_count which doesn’t take into a could cgroups or affinity restrictions to limit the amount of available CPU. It’s a big issue on the grid, in binder and in other HPC environments where machines have 128+ cores but only a few allocated to the current job.

See https://bugs.python.org/issue26692